ui/src/app/components/gke-app-configuration-panels/create-gke-app.tsx

```diff
@@ -215,6 +220,23 @@ export const CreateGkeApp = ({
   const showDeleteDiskButton = unattachedDiskExists(app, disk);
   const showDeleteAppButton = canDeleteApp(app);
 
+  const canModifyDiskSize = !!app || unattachedDiskExists(app, disk);
```

Comment:
this is not right.  These are the conditions where you can't modify the disk size.